check.js

```diff
@@ -1,6 +1,9 @@
 var fs = require('fs');
 
 const disabledForMachine = process.env.npm_package_config_disabled;
+const greetingLogged = process.env.npm_package_config_logged;
+const optoutLogged = process.env.npm_package_config_optout;
```

Comment:
This would be less confusing is named `optoutLogged` or similar. Now it is easy to confuse it with `disabled` (which actually means that the user has optedout)

Reply:
Changed.